src/cmd_say.rs

```diff
@@ -0,0 +1,100 @@
+            --==---===---::::::::::::::::-----------=====+  
+            -------+**+----------------------------====***  
+            ---------------::::::::::------------======#**  
+            -----=+++++-----------------=-=--=---======*+=  
+            -----=+++++--#@@@%%%%###+---=-=--+---======:.   
+            .......::----+####%%%%@@*---++++++---===:.      
+                  .*########*:.......:==--------.         
+                :*#%%%%%%%%%%+       -%######*#+.         
+                =#########%%%+     =#%%%%%%%%%##-         
+                -++***#####=.      *############:         
+                                   -==++***##+:           
+";
+            writeln!(ctx.io.out, "{}", border).ok();
+            writeln!(ctx.io.out, "{}", print_text).ok();
+            writeln!(ctx.io.out, "{}", border).ok();
+            writeln!(ctx.io.out, "{}", kitty_string).ok();
+        } else {
+            kitty_string = "                                                  
+                    .....                                 
+                   .::-:...            .....              
+                  ..:---..:...        .::::...            
+                 ..------:.::::::::::.:----......         
+              .::::------:::::::::::..------:..::::::::-. 
+            .::::..........::::::::::::::----:::::::::---. 
+            ::::::::::::::::::::::::...........::::::::---. 
+            :--:::::::::::::::::::::::::::::::::::::::----. 
+            :--::=#@@@%%%###***+++===---::::::::--::-=----. 
+            :--::#@@@@@@@@@@@@@@@@@@@@@@@@@#-:::---:=-=---. 
+            :--::#@@@@@@@@@@@@@@@@@@@@@@@@@@@:::----++=---. 
+            :--::#@@@@%***#@@@@@@@@@*+*@@@@@@:::----=+----. 
+            :--::#@@@**%%%#+@@@@@@@@=-=@@@@@@:::----------  
+            :---:#@@@@@@@@@@@%%%%@@@=-=@@@@@@:::---------=  
+            -----#@@@@@@@**@@#+-+#@@#%%@@@@@@:::--------==  
+            -----#@@@@@@@@%+#%#-%@%+*#@@@@@@@::--------===  
+            -----*%@@@@@@@@@***+++*%@@@@@@@@@----------===. 
+            ------=+***####%%%@@@@@@@@@@@@@@@---------====. 
+            ----------::::::::::::--===+++*+:--------=====  
+            --==---===---::::::::::::::::-----------=====+  
+            -------+**+----------------------------====***  
+            ---------------::::::::::------------======#**  
+            -----=+++++-----------------=-=--=---======*+=  
+            -----=+++++--#@@@%%%%###+---=-=--+---======:.   
+            .......::----+####%%%%@@*---++++++---===:.      
+                  .*########*:.......:==--------.         
+                :*#%%%%%%%%%%+       -%######*#+.         
+                =#########%%%+     =#%%%%%%%%%##-         
+                -++***#####=.      *############:         
+                                   -==++***##+:           
+";
+            writeln!(ctx.io.out, "{}", kitty_string).ok();
+        }
+        Ok(())
+    }
+}
```

Comment:
We should add at least one test. Check https://github.com/KittyCAD/cli/blob/main/src/cmd_file.rs#L431 and other files for some example test setup.